### ISSUE TITLE
Plugin added but cannot access

### ISSUE THREAD

**Issue by waltuy** [kolsys/YouTubeTV.bundle#76]
I have installed Plex Media Server version 1.18.7.2438 in docker on Ubuntu. I followed your instructions to load the plugin into Plex directory. The plugin did show up in my plex but when I hover my mouse over it I can click only on settings or delete icon on it. Can someone please point out to me if I am doing something wrong, I had restarted plex and rebooted my server as well but no luck.

Please refer screenshots below:
https://storage.matiboux.com/imgshot/h2NFWiF2ZZKk.jpg
https://storage.matiboux.com/imgshot/sn7C3vqIbFHm.jpg

Thanks in advance.

**Comment by pedropilla** [comment]
> I have installed Plex Media Server version 1.18.7.2438 in docker on Ubuntu. I followed your instructions to load the plugin into Plex directory. The plugin did show up in my plex but when I hover my mouse over it I can click only on settings or delete icon on it. Can someone please point out to me if I am doing something wrong, I had restarted plex and rebooted my server as well but no luck.
> 
> Please refer screenshots below:
> https://storage.matiboux.com/imgshot/h2NFWiF2ZZKk.jpg
> https://storage.matiboux.com/imgshot/sn7C3vqIbFHm.jpg
> 
> Thanks in advance.

Me too.

**Comment by anatosun** [comment]
Same here

**Comment by vincegre** [comment]
Found how to use the plugin ??

**Comment by feronetick** [comment]
As i know plex no more support plugins. At least on new versions. Can anyone know the specific version of server (client) it support this

**Comment by PedroCavaleiro** [comment]
Anyone found the solution to this?

**Comment by q-t-p** [comment]
Same here on Synology. The plugin shows up, but just the settings screen is working.

**Comment by NicoSoler** [comment]
Pretty much this. Installed the plugin, it appears on Settings > Plugins, but I cannot use it.

**Comment by vincegre** [comment]
> Pretty much this. Installed the plugin, it appears on Settings > Plugins, but I cannot use it.

PLugins are deprecated in Plex since ages so don't expect that to work anymore....

**Comment by ghost** [comment]
> > Pretty much this. Installed the plugin, it appears on Settings > Plugins, but I cannot use it.
> 
> PLugins are deprecated in Plex since ages so don't expect that to work anymore....

Yes and no. Some plugins do still work but they are mainly metadata scrapers. This one definetly does not work though as it does lot more than just scrape metadata. I would suggest switching over to Jellyfin if having Youtube inside your media server is important. 
https://github.com/ankenyr/jellyfin-youtube-metadata-plugin
